test/schema-feature.test.js

```diff
@@ -999,9 +1040,10 @@ test('Check how many AJV instances are built #2 - verify validatorPool', t => {
   fastify.register(function sibling1 (instance, opts, next) {
     addRandomRoute(instance)
     t.notOk(instance.validatorCompiler, 'validator not initialized')
-    instance.ready(() => {
+    instance.addHook('onReady', function one (done) {
```

Comment:
@mcollina I had to change this test because if the user calls `instance.ready()` multiple time, the order of execution is switched (from function one, two, three to three, two, one)

I'm investigating